packages/strapi-design-system/src/SubNav/SubNavLink.js

```diff
@@ -88,5 +88,6 @@ SubNavLink.propTypes = {
   icon: PropTypes.element,
   isSubSectionChild: PropTypes.bool,
   link: PropTypes.element,
+  subSectionChild: PropTypes.bool,
```

Comment:
should it be `isSubSectionChild`?

Reply:
Good catch. I messed up the rebease - should be fixed.